iconservice/icon_service_engine.py

```diff
@@ -581,13 +583,31 @@ def _before_transaction_process(self,
         self._update_productivity(context, prev_block_generator, prev_block_votes)
         self._update_last_generate_block_height(context, prev_block_generator)
 
+    @classmethod
+    def _put_block_produce_info_on_start_calc(cls,
+                                              context: 'IconScoreContext',
+                                              prev_block_generator: Optional['Address'] = None,
+                                              prev_block_votes: Optional[List[Tuple['Address', bool]]] = None):
+
```

Comment:
Need to consider the genesis decentralization. At that time, do not have to put bp info as prevote information.